### PR TITLE
Support for Node v4

### LINKED ISSUE #32: Port to Node.JS 4.x
Noted here, this package is currently failing to build under Node.JS 4.x.

The upstream issue to track: https://github.com/nodejs/node/issues/2798

Discovered this while getting MeanJS to 4.x, see https://github.com/meanjs/mean/pull/922

### PR DESCRIPTION
**Pull request by lirantal**
Bumping node-inspector version to latest to provide support for node v4
Closes #32

### CODE REVIEW

**Comment by flexpert** [comment]
+1 for that! 
Great @lirantal

**Comment by erg0dic** [comment]
+1
Would love to see this merged in to fix downstream modules

**Comment by lirantal** [comment]
Thanks Robert, I hope @ChrisWren can address this issue ASAP

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "grunt-node-inspector",
-  "version": "0.2.0",
+  "version": "0.2.1",
```
Comment (ChrisWren) [inline package.json]: Could you remove this diff and rebase to a single commit. Then I will merge and bump the version myself.

**Comment by lirantal** [comment]
@ChrisWren done

**Pull request merged**